aicsimageio/readers/czi_reader.py

```diff
@@ -363,16 +363,14 @@ def dtype(self) -> np.dtype:
     @property
     def metadata(self) -> _Element:
         """
-        Lazy load the metadata from the CZI file
+        Load and return the metadata from the CZI file
 
         Returns
         -------
-        The xml Element Tree of the metadata
+        The lxml Element Tree of the metadata
         """
-        if self._metadata is None:
-            # load the metadata
-            self._metadata = CziFile(self._file).meta
-        return self._metadata
+        # We can't serialize lxml element trees so don't save the tree to the object state
+        return CziFile(self._file).meta
```

Comment:
Good note, but how expensive is this operation? I don't imagine it to be very heavy, just curious if we know how much it costs for a larger file.

Reply:
Yea it's hard to say how heavy this is... I don't think it's that bad. Even for something like a 2TB file I can't imagine it would be more than 2 seconds ish? disregarding transfer times